charts/posthog/templates/pgbouncer-hpa.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.pgbouncer.hpa.enabled -}}
```

Comment:
Will this work? Postgres has pretty strict connection limits internally and each pgbouncer takes up N connections - scaling pgbouncers out of bounds can easily lead to running out of connections -> errors & outages.